cmd/pro/add/add.go

```diff
@@ -0,0 +1,18 @@
+package add
+
+import (
+	proflags "github.com/loft-sh/devpod/cmd/pro/flags"
+	"github.com/spf13/cobra"
+)
+
+// NewAddCmd creates a new command
+func NewAddCmd(globalFlags *proflags.GlobalFlags) *cobra.Command {
+	addCmd := &cobra.Command{
+		Use:   "add",
+		Short: "Adds a cluster to DevPod Pro",
```

Comment:
`add` itself doesn't add a cluster, `add cluster` adds a cluster to devpod pro. Maybe `Adds given resource to DevPod Pro` or something along those lines?

Reply:
makes sense 👍